internal/base/base.go

```diff
@@ -29,36 +29,57 @@ import (
 	"beanq/internal/options"
 )
 
-func makeKey(group, queue, name string) string {
+func makeKey(keys ...string) string {
+
+	var builder strings.Builder
+
+	for _, v := range keys {
+		builder.WriteString(v)
+	}
+	return builder.String()
```

Comment:
Instead of parsing ":" as part of `keys` params every time, how about we just use String.Join(keys, ":") instead to make the params simpler?

Reply:
because there will also be other strings.
for example:
`makeKey(prefix, ":", group, ":", queue, ":", "list")`

Reply:
yes,  strings.Join(keys,":")   is better.